File: Stage7/Python/Problem2566.py
def main():
    matrix = [list(map(int, input().split())) for _ in range(9)]
    
    value, row, col = -1, 0, 0

    for i in range(9):
        for j in range(9):
            if matrix[i][j] > value:
                value = matrix[i][j]
                row = i + 1
                col = j + 1
        
    print(value)
    print(row, col)
    
if __name__ == "__main__":
    main()

File: Stage7/C/Problem2566.c
#include <stdio.h>
#include <limits.h>

int main() {
    int matrix[9][9];
    int row = 0, col = 0, value = INT_MIN;

    for (int i = 0; i < 9; i++) {
        for (int j = 0; j < 9; j++) {
            scanf("%d", &matrix[i][j]);
        }
    }

    for (int i = 0; i < 9; i++) {
        for (int j = 0; j < 9; j++) {
            if (matrix[i][j] > value) {
                value = matrix[i][j];
                row = i + 1;
                col = j + 1;
            }
        }
    }

    printf("%d\n%d %d", value, row, col);
    return 0;
}